### ISSUE TITLE
With two different data sets I am getting exactly same (identical) probabilities for the flow between the locations from Radiation()

### ISSUE THREAD

**Issue by rejusam** [scikit-mobility/scikit-mobility#183]
In the following code for the radiation model:
[https://github.com/scikit-mobility/scikit-mobility/blob/master/skmob/models/radiation.py](https://github.com/scikit-mobility/scikit-mobility/blob/master/skmob/models/radiation.py)

I am unable to locate how the keyword `out_format='probabilities'` works.
Also, with two different data sets I am getting exactly same (identical) probabilities for the flow between the locations, which is quite suspicious.
Could you please have a look into this issue?

Thank you,
Reju

**Comment by FilippoSimini** [comment]
Could you please provide the code of a short example to reproduce your issue?

**Comment by rejusam** [comment]
@FilippoSimini 
The code: 

> from skmob.models import Radiation
> np.random.seed(0)
> radiation = Radiation()
> rad_flows = radiation.generate(tessellation, tile_id_column='tile_ID', tot_outflows_column='tot_outflows',  
>                                relevance_column='population', out_format='probabilities') #random sample of flows
> display(rad_flows.head())

![Screenshot 2021-06-02 at 10 51 59 AM](https://user-images.githubusercontent.com/6176357/120399679-99ebb380-c390-11eb-8921-5c0ef4394751.png)
on a particular day's data set  is producing the exactly same (identical) probabilities  for the flow between the locations for another day's data set.
Probabilities of flow with second data set:  
![Screenshot 2021-06-02 at 11 04 50 AM](https://user-images.githubusercontent.com/6176357/120400655-627e0680-c392-11eb-9dc8-d73e1f44fbfd.png)

First data set:
![Screenshot 2021-06-02 at 11 00 37 AM](https://user-images.githubusercontent.com/6176357/120400385-e1267400-c391-11eb-88e6-4c0792267ef0.png)

Second data set:
![Screenshot 2021-06-02 at 11 02 03 AM](https://user-images.githubusercontent.com/6176357/120400444-ff8c6f80-c391-11eb-87d1-eb3456b66306.png)

**Comment by rejusam** [comment]
@FilippoSimini Thanks for the clarification .... :-)

**Comment by rejusam** [comment]
Dear Filippo,
This may be a silly question. But I would like to know, can we use any
other information other than population to the relevance_column?
If yes, what will be the effect of feeding this new value to the
relevance_column?

Thank You,
Reju

**Comment by FilippoSimini** [comment]
Yes, the relevance column can be any appropriate (positive) quantity to characterize a location's attractiveness for the specific flows to be generated: the higher a location's relevance value, the more probable will be to observe a flow to that location.

**Comment by rejusam** [comment]
Thank You very much... :-)
